clang/lib/Parse/ParseOpenMP.cpp

```diff
@@ -4675,8 +4709,10 @@ bool Parser::ParseOpenMPVarList(OpenMPDirectiveKind DKind,
     // Only parse map-type-modifier[s] and map-type if a colon is present in
     // the map clause.
     if (ColonPresent) {
+      if (getLangOpts().OpenMP >= 60 && getCurToken().is(tok::colon))
```

Comment:
```suggestion
      if (getLangOpts().OpenMP >= 60)
```

Reply:
ColonPresent is already true.

Reply:
I only need to emit error for map(: a),   I don't need emit error like map(,,: a), since the error already emit during parseMapTypeModifiers. 

The ColonPresent will be set for when ":" exist for map clause.

Reply:
Maybe emit then the error in parseMapTypeModifiers too?

Reply:
parseMapTypeModifiers is recursive function.  I don't I have good place to added that.